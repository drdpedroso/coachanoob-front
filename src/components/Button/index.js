import React from 'react'
import PropTypes from 'prop-types';
import {Button} from 'semantic-ui-react'

const BaseButton = (props) => {
    return (
        <Button
            {...props}
            onClick={props.onClick}
        >
            {props.children}
        </Button>
    )
};

BaseButton.propTypes = {
    onClick: PropTypes.func,
    children: PropTypes.node
};

export default BaseButton
