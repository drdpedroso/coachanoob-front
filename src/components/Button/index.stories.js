import React from 'react';
import {storiesOf} from '@storybook/react';
import {action} from '@storybook/addon-actions';

import Button from './';
import {Icon} from "semantic-ui-react";

storiesOf('Button', module)
    .add('with text primary', () => (
        <Button primary onClick={action('clicked')}>Hello Button</Button>
    ))
    .add('with text secondary', () => (
        <Button secondary onClick={action('clicked')}>Hello Button</Button>
    ))
    .add('with icon primary', () => (
        <Button primary onClick={action('clicked')}><Icon name='world'/></Button>
    ))
    .add('with icon secondary', () => (
        <Button secondary onClick={action('clicked')}><Icon name='world'/></Button>
    ))
    .add('with texxt primary fixed bottom', () => (
        <Button attached='bottom' onClick={action('clicked')}>Hello Button</Button>
    ))
    .add('with text primary fluid', () => (
        <Button fluid onClick={action('clicked')}>Hello Button</Button>
    ));
