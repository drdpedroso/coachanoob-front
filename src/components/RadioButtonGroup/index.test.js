import React from 'react';
import Button from './index'
import renderer from 'react-test-renderer'
import {render} from ''

it('renders correctly', () => {
    const tree = renderer
        .create(<Button>Menor Bolado</Button>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
