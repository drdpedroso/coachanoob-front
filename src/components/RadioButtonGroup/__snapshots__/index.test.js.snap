// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButtonGroup renders correctly 1`] = `
<div
  className="ui secondary sc-bwzfXH dyjRzU menu"
>
  <a
    className="item sc-bdVaJa vEBRA"
    data-testid="1+"
    onClick={[Function]}
  >
    1+
  </a>
  <a
    className="item sc-bdVaJa vEBRA"
    data-testid="2+"
    onClick={[Function]}
  >
    2+
  </a>
  <a
    className="item sc-bdVaJa vEBRA"
    data-testid="3+"
    onClick={[Function]}
  >
    3+
  </a>
  <a
    className="item sc-bdVaJa vEBRA"
    data-testid="4+"
    onClick={[Function]}
  >
    4+
  </a>
  <a
    className="item sc-bdVaJa vEBRA"
    data-testid="5+"
    onClick={[Function]}
  >
    5+
  </a>
</div>
`;
